pkg/apis/serving/v1beta1/predictor_pmml.go

```diff
@@ -58,11 +56,8 @@ func (p *PMMLSpec) GetContainer(metadata metav1.ObjectMeta, extensions *Componen
 		fmt.Sprintf("%s=%s", constants.ArgumentModelDir, constants.DefaultModelLocalMountPath),
 		fmt.Sprintf("%s=%s", constants.ArgumentHttpPort, constants.InferenceServiceDefaultHttpPort),
 	}
-	if !utils.IncludesArg(p.Container.Args, constants.ArgumentWorkers) {
-		if extensions.ContainerConcurrency != nil {
-			arguments = append(arguments, fmt.Sprintf("%s=%s", constants.ArgumentWorkers, strconv.FormatInt(*extensions.ContainerConcurrency, 10)))
-		}
-	}
+	p.Container.Args = utils.RemovePrefix(p.Container.Args, constants.ArgumentWorkers) // The PMMLServer doesn't support multiple workers.
```

Comment:
I think we should reject `--worker` if not setting to 1 in the `Validate` function above instead of silently removing the argument

Reply:
Ok, I've already done it.